lib/utility/es_client.rb

```diff
@@ -13,7 +13,7 @@ module Utility
   class EsClient < ::Elasticsearch::Client
     USER_AGENT = 'elastic-web-crawler-'
     MAX_RETRIES = 3
-    REQUEST_TIMEOUT = 30 # seconds
+    REQUEST_TIMEOUT = 10 # seconds
```

Comment:
30 second timeout with exponential backoff made for some very long wait times, I think 10 seconds for an ES timeout is enough for default (users can also change this value if they need)